low-level/framework/rabbitmq/rabbitmq_egress_processor.py

```diff
@@ -307,7 +310,7 @@ def _transmit_msg_on_exchange(self):
                 self._event.set()
 
         except Exception as ex:
-            logger.error("RabbitMQegressProcessor, _transmit_msg_on_exchange: %r" % ex)
+            logger.error(f'RabbitMQegressProcessor, _transmit_msg_on_exchange, problem while publishing the message:{ex}, adding message to consul: {self._jsonMsg}')
```

Comment:
Here, I do not see the code for adding message to consul. Still I will go ahead and merge this PR since changes at line #302 to #304 should help with the issue seen in testing.
@Madhura-08 Please check this and fix the same through separate PR, if needed.

Reply:
@indrajitzagade you only raised this earlier that storing to consul for any generic exception during the course of this function not needed and may cause unintended effects. Me, @sandeepsgt agreed and there store.put() was removed by @Madhura-08 .So i believe we are good here

Reply:
@cdeshmukh @Madhura-08 Yes, I had raised it earlier. And good that we all agree on the same.
But then the logger here is misleading, which is why I got confused.

Reply:
Oh right, she missed removing storing to consul comment. @Madhura-08 please make note and update through any future patch, may be ask someone whose next patch is coming, when see in EOS-RAS channel.